cli/src/config/templates.toml

```diff
@@ -312,20 +339,26 @@ commit_summary_separator = 'label("separator", " | ")'
 'format_short_operation_id(id)' = 'id.short()'
 'format_path(path)' = 'path.display()'
 'format_short_signature(signature)' = '''
-  coalesce(signature.email(), email_placeholder)'''
+coalesce(signature.email(), email_placeholder)
+'''
 'format_short_signature_oneline(signature)' = '''
-  coalesce(signature.email().local(), email_placeholder)'''
+coalesce(signature.email().local(), email_placeholder)
+'''
 
 'format_detailed_signature(signature)' = '''
-  coalesce(signature.name(), name_placeholder)
-  ++ " <" ++ coalesce(signature.email(), email_placeholder) ++ ">"
-  ++ " (" ++ format_timestamp(signature.timestamp()) ++ ")"'''
+separate(" ",
+  coalesce(signature.name(), name_placeholder),
+  surround("<", ">", coalesce(signature.email(), email_placeholder)),
+  surround("(", ")", format_timestamp(signature.timestamp())),
```

Comment:
nit: no need to use `surround()` here? It's technically correct that `format_timestamp()` could be an empty, but that's unlikely.

Reply:
I was using `surround()` so that it's easier to see that `content` is being surrounded with `()` (and similarly `<>` for the email). I wasn't depending on the fact that `surround()` does nothing if `content` is empty.

Reply:
I'll merge in ~24 hours if no other objections, but I can revert using `surround()` if you want.